### ISSUE TITLE
Product ID ProPlus2019Retail is missing

### ISSUE THREAD

**Issue by msvamp** [MicrosoftDocs/OfficeDocs-Support#558]
In the list of _non-Office 365 product IDs supported by the ODT_, the `ProPlus2019Retail` product ID is **missing** although it is a valid product ID supported by the Office Deployment Tool - tested this on the latest ODT (version 12624.20320 at the time of writing).

For example, the below configuration successfully installed the latest Office 2019 Professional Plus Retail edition en-us with the Broad Update channel (Build 16.0.11929.20776 at the time of writing) :-

```XML
<Configuration>
  <Add OfficeClientEdition="64" Channel="Broad" >
   <Product ID="ProPlus2019Retail" >
     <Language ID="en-us" />
   </Product>
 </Add>
 <Display Level="Full" AcceptEULA="FALSE" />
 <Updates Enabled="TRUE" Channel="Broad" />
</Configuration>
```

There is another issue MicrosoftDocs/OfficeDocs-DeployOffice#225 which mentions installing Office 2019 Retail edition using the ODT.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 358c3fd4-27aa-fc10-e23e-43b3f801b739
* Version Independent ID: 3d322857-f041-5b22-d91b-458427e45fb1
* Content: [Product IDs that are supported by the Office Deployment Tool for Click-to-Run - Office 365](https://docs.microsoft.com/en-in/office365/troubleshoot/installation/product-ids-supported-office-deployment-click-to-run)
* Content Source: [Microsoft365/admin/installation/product-ids-supported-office-deployment-click-to-run.md](https://github.com/MicrosoftDocs/OfficeDocs-Support/blob/public/Microsoft365/admin/installation/product-ids-supported-office-deployment-click-to-run.md)
* Service: **o365-solutions**
* GitHub Login: @AmandaAZ
* Microsoft Alias: **v-todmc**

**Comment by Reeced40** [comment]
@msvamp thank you for reaching out and helping improve MS Docs. I see this is reflecting in the article now. Please confirm from your side and let me know if you think there is anything else missing from the article.

**Comment by msvamp** [comment]
Yes, I believe this was reported again in another issue #1266 which was closed by the PR #1287.
